DEVELOPER.md

```diff
@@ -176,6 +176,12 @@ $ ./k8-util/cluster/reset-kind.sh
 #### Build Fluvio CLI and docker image from source code
 
 ```
+# Install the cross-compilation toolchain needed to compile the docker image (only required the first time)
+# x86_64 (most computers):
+$ rustup target add x86_64-unknown-linux-musl
+# M1 Macs:
+$ rustup target add aarch-unknown-linux-musl
```

Comment:
I think that it is `aarch64-unknown-linux-musl`

Reply:
Oh shoot my bad. I'll fix it when I get home, thanks.

Reply:
Fixed